samples/sdl_joystick_gamepad/Makefile

```diff
@@ -0,0 +1,10 @@
+DEBUG = y
+XBE_TITLE = sdl_joystick_gamepad
```

Comment:
This sample should be `sdl_joystick_gamecontroller` to reflect what it actually does.
It could also just be `sdl_gamecontroller` (recommended) as that's the API prefix.